js/index.js

```diff
@@ -1,3 +1,32 @@
+const nameElem = document.getElementById('user-name');
+const emailElem = document.getElementById('user-email');
+const messageElem = document.querySelector('#user-message');
+
+function setInfo() {
+  const currentName = localStorage.getItem('formInfo').split(',')[0];
+  nameElem.value = currentName;
+  const currentEmail = localStorage.getItem('formInfo').split(',')[1];
+  emailElem.value = currentEmail;
+  const CurrentMessage = localStorage.getItem('formInfo').split(',')[2];
+  messageElem.value = CurrentMessage;
+}
+
+function populateStorage() {
+  const info = [nameElem.value, emailElem.value, messageElem.value].join();
```

Comment:
- [ ] Kindly be noticed that the form data for all inputs should be stored in a single object.

```javascript
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```